locales/he_IL/LC_MESSAGES/duckduckgo.po

```diff
@@ -3469,18 +3480,22 @@ msgstr "ניקוד"
 msgid "Scroll down and click %sView advanced settings%s"
 msgstr "גלול/י מטה והקש/י %sצפה/י בהגדרות מתקדמות%s"
 
+#, fuzzy
+#| msgid "Scroll down and click %sView advanced settings%s"
 msgid "Scroll down and click %sView advanced settings%s."
-msgstr ""
+msgstr "גלול/י מטה והקש/י %sצפה/י בהגדרות מתקדמות%s"
 
 msgid ""
 "Scroll down and find %sSearch in the address bar%s. Click on %sChange%s (or "
 "%sAdd new%s)"
 msgstr ""
 
+#, fuzzy
+#| msgid "Under %sSearch in the address bar with%s select %sAdd New%s"
 msgid ""
 "Scroll down and find %sSearch in the address bar%s. Click on %sChange%s (or "
 "%sAdd new%s)."
-msgstr ""
+msgstr "מתחת %sחפש בשורת הכתובת עם%s בחר %sהוסף חדש%s"
```

Comment:
Here we should have 3 snippets surrounded by placeholder tags, for a total of 6 tags, but I only see 4 in the translation (2 snippets). We need all placeholder tags to be in place, otherwise the system will break. Do you mind taking a look?

Reply:
Fixed this too.